lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -4,6 +4,9 @@ module CLI
   module UI
     module Prompt
       class InteractiveOptions
+        DONE = "Done"
+        CHECKBOX_ICON = { false => "⬡", true => "⬢" }
```

Comment:
Could we use ```CHECKBOX_ICON = { false => "☐", true => "☑" } ``` instead?  The hexagons look too much like circles => radio boxes => single selection.

Reply:
The box is a little small though:
![image](https://user-images.githubusercontent.com/1216046/45563082-40c34680-b81a-11e8-986e-2308b0b6c030.png)

What do you think?

Reply:
Not married to that pair in particular - open to other options, but I think it's better than something that looks like single selection.

Reply:
I couldn't find a better option, so I went with the checkbox.